paddle/fluid/pir/dialect/operator/ir/ops.yaml

```
@@ -1565,6 +1565,16 @@
   kernel:
     func: onednn_to_paddle_layout
 
+- op: partial_send
+  args: (Tensor x, int ring_id = 0, int peer = 0, bool use_calc_stream = false, int num = 1, int id = 0)
+  output: Tensor(out)
```

```suggestion
  output: 
```
这个算子是没有输出的，其他地方也需要相应修改